src/Crash.Handlers/Utils/RhinoObjectAttributesUtils.cs

```diff
@@ -0,0 +1,191 @@
+﻿using Rhino.DocObjects;
+
+namespace Crash.Handlers
+{
+	public static class RhinoObjectAttributesUtils
+	{
+		// TODO : Move
+		internal static Dictionary<string, string> GetAttributeDifferencesAsDictionary(ObjectAttributes oldAttributes,
```

Comment:
This is very ugly. Definetly needs to be improved. Fine to do it in a separate PR

Reply:
Agreed haha